frontend/src/scenes/settings/environment/ReplayTriggers.tsx

```diff
@@ -671,12 +676,25 @@ function TriggerMatchChoice(): JSX.Element {
                     }}
                     value={currentTeam?.session_recording_trigger_match_type_config || 'all'}
                 />
-                <div>triggers match</div>
+                <div>triggers below match</div>
             </div>
         </div>
     )
 }
 
+function TriggerMatchTypeTag(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    // Let's follow PostHog style of AND / OR from funnels
+    return (
+        <LemonTag type="danger" className="my-2 mr-2">
+            {currentTeam?.session_recording_trigger_match_type_config &&
+            currentTeam?.session_recording_trigger_match_type_config === 'any'
+                ? 'OR'
+                : 'AND'}
+        </LemonTag>
+    )
+}
```

Comment:
ambivalent about this... mostly because there's already so much text

but no strong opinion and will rework this area as part of triggers 3000 so worth seeing how folk respond to it